### ISSUE TITLE
syslog appender

### ISSUE THREAD

**Issue by sathishtb** [Adobe-Consulting-Services/acs-aem-commons#1461]
## Required Information

* [x] AEM Version, including Service Packs, Cumulative Fix Packs, etc: AEM 6.3
* [x] ACS AEM Commons Version:  acs-aem-commons-content-3.17.0
* [x] Reproducible on Latest? yes

### Expected Behavior

I have installed latest acs-aem-commons package, and configured syslog server details(IP&Port) for ACS AEM Commons - Syslog Appender, but logs are not sending to syslog server.

How to activate syslog appender? After configuring host and port in SyslogAppender in OSGi, syslog will be activated automatically?

**Comment by justinedelson** [comment]
As described on https://adobe-consulting-services.github.io/acs-aem-commons/features/syslog-appender/index.html, it depends on what you put as the logger names.

**Comment by sathishtb** [comment]
Thanks for your reply.

Above link gives instruction how to config the syslog appender.

I have installed latest acs-aem-commons package, and configured syslog server details(IP&Port) for "ACS AEM Commons - Syslog Appender" , and configured logger name as ROOT to log all messages.but logs are not sending to syslog server.

How to make to send all log messages to syslog server?

**Comment by justinedelson** [comment]
That should be it. I just did a test with the latest release using the syslog support on a Synology Diskstation and it worked as expected

Here's the configuration in Diskstation:
![image](https://user-images.githubusercontent.com/65906/44299070-146ce680-a2bd-11e8-8fb3-7d15bc96f620.png)

Here's the AEM configuration:
![image](https://user-images.githubusercontent.com/65906/44299082-34040f00-a2bd-11e8-8625-1b6c58fc77a2.png)

And here's a few log messages in the Diskstation Web UI:
![image](https://user-images.githubusercontent.com/65906/44299094-56962800-a2bd-11e8-8448-70598c42485a.png)

My guess is that some combination of one or more of these things are happening:
1. Your syslog service is only accepting TCP connections (Logback Syslog only supports UDP)
2. There is a network restriction in place.
3. The syslog service doesn't accept remote connections

I added a note to the documentation about UDP being required. There was an issue to add TCP support (#1264), but there are licensing issues related to this.
